src/utils/ensure-js-imports.test.js

```diff
@@ -48,9 +48,10 @@ test('ensureJsImports before', () => {
   const path = parseJs(`const a = 1;`);
   const vars = ensureJsImports(path, `import foo, {bar} from 'bar';`);
```

Comment:
minor: could leave the semicolon off one of these so the test validates adding it

Reply:
there are other tests in the file which don't include the semi, just not visible in this diff